api/src/Microsoft.Azure.IIoT.Api/src/Publisher/Models/EngineConfigurationApiModel.cs

```diff
@@ -43,10 +43,10 @@ public class EngineConfigurationApiModel {
         public int? MaxMessageSize { get; set; }
 
         /// <summary>
-        /// Max IoT D2C outgress message buffer size
+        /// Max IoT D2C egress message queue
         /// </summary>
-        [DataMember(Name = "maxOutgressMessages", Order = 4,
+        [DataMember(Name = "maxEgressMessages", Order = 4,
```

Comment:
Please use the same name for data member as for class member, i.e. `maxEgressMessageQueue`.

Reply:
Fixed